analysis_brokers.tex

```diff
@@ -0,0 +1,8 @@
+\textbf{B-201} Identify Tidal Disruption Events (TDEs) and extreme AGN by constructing long-term light curves from difference images using forced photometry.
```

Comment:
These are again too specific. The things brokers offer that the RSP doesn't is real-time streaming and filters, LC classification, watchlists notifications, etc. Focus on that functionality. I can help here.

Reply:
Confirmed resolved.